OracleTuxedo/OracleTSAM/dockerfiles/12.2.2/files/sbin/init.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+source ~/sbin/functions.sh
+source /etc/envs/bashrc
+
+nohup sudo /sbin/sshd -D >/dev/null 2>&1 &
```

Comment:
There is no `init` inside a Docker container, so you need to be careful about child processes you spawn without reaping. By `nohup`'ing `'sudo` you make it more difficult to introspect the process inside the container or reap it properly.